### PR TITLE
SC-6831 - fixes search icon behavior on lernstore

### PR DESCRIPTION
**Pull request by JenyMzo**
# Description

Changes search icon behaviour so it only shows the magnifying glass when the input is empty, otherwise it shows the clear icon to reset the search.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-6831

## Changes

- Disables click on search button (now is only informative)
- Changes search/clear icons behaviour

## Screenshots of UI changes

Screenshot 2020-10-12 at 09 43 06
Screenshot 2020-10-12 at 09 42 52

## Approval for review

- [X] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by JenyMzo** [comment]
@schul-cloud-bot update screenshots

**Review by virgilchiriac — APPROVED** [review]

**Comment by virgilchiriac** [comment]
I guess changelog should also be updated

**Review by virgilchiriac — DISMISSED** [review]

File: src/components/molecules/ContentSearchbar.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
 				@focus="isActive = true"
 			/>
 			<div class="search__container--icon">
-				<base-button v-if="isActive" design="none" @click="enterKeyHandler">
+				<base-button v-if="value === ''" design="none">
```
Comment (adrianjost) [inline src/components/molecules/ContentSearchbar.vue:17]: having a button without a click handler does not make sense semantically.

**Comment by JenyMzo** [comment]
@schul-cloud-bot update screenshots

**Review by virgilchiriac — APPROVED** [review]

**Review by CordlessWool — APPROVED** [review]

**Pull request merged**